test/lib.bash

```diff
@@ -45,7 +45,49 @@ function run_e2e_tests {
     && return 1
 }
 
+function run_upstream_tests {
+  (
+  # Checkout the relevant code to run
+  mkdir -p "$GOPATH/src/knative.dev"
+  cd "$GOPATH/src/knative.dev" || exit
+  #git clone -b "release-v$1" --single-branch https://github.com/openshift/knative-serving.git serving
+  git clone -b "ocp-flag-0.9.0" --single-branch https://github.com/markusthoemmes/knative-serving.git serving
```

Comment:
We should probably unify how we consume other repositories. For example, in this other PR we use Go dependencies: https://github.com/openshift-knative/serverless-operator/pull/32/files#diff-836546cc53507f6b2d581088903b1785R41

Reply:
Discussed this with @cardil and landed on the current code as a way forward.